<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_ModifyMirrorOptions</class>
 <widget class="QWidget" name="LC_ModifyMirrorOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbMirrorToLine">
     <property name="toolTip">
      <string>If checked, mirror will be performed by selected line entity.</string>
     </property>
     <property name="text">
      <string>Mirror to selected line</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cbKeepOriginals">
     <property name="toolTip">
      <string>If unchecked, original entities will be removed, otherwise they will survive.</string>
     </property>
     <property name="text">
      <string>Keep Originals</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cbCurrentAttr">
     <property name="toolTip">
      <string>If checked, current attributes will be used for mirrored entities, otherwise - original ones. </string>
     </property>
     <property name="text">
      <string>Use Current Attributes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="cbLayer">
     <property name="toolTip">
      <string>If checked, mirrored entities will be placed on current layer, otherwise they will be placed into original layers.</string>
     </property>
     <property name="text">
      <string>User Current Layer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
